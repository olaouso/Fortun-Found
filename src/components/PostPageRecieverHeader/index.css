.recieverHeaderImage {
  width: 100%;
  height: auto;
}

.postHeaderRecieverText p {
  display: flex;
  align-items: center;
  font-family: "Acme";
  color: #590995;
  font-size: 3em;
  text-align: center;
  padding: 5%;
}

.postRecieveHeader {
  margin-bottom: 2%;
  margin-top: 2%;
}
