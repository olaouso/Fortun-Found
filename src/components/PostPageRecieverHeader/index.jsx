/* eslint-disable max-len */
/* eslint-disable react/self-closing-comp */
import React from "react";
import { Row, Col, Container } from "react-bootstrap";
import { useTranslation } from "react-i18next";
import recieverHeaderPic from "./image/recieverHeaderPic.jpg";
import "./index.css";

export default function PostPageRecieverHeader() {
  const { t } = useTranslation();

  return (
    <Container fluid="md" className="postRecieveHeader">
      <Row>
        <Col md={6}>
          <img
            className="recieverHeaderImage"
            src={recieverHeaderPic}
            alt={t("postPageReciever.altRecieverHeader")}
          />
        </Col>
        <Col className="postHeaderRecieverText">
          <p>{t("postPageReciever.title")}</p>
        </Col>
      </Row>
    </Container>
  );
}
