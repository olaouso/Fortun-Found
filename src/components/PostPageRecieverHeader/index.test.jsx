import React from "react";
import renderer from "react-test-renderer";
import PostPageRecieverHeader from "./index";

it("renders PostPageRecieverHeader component correctly", () => {
  const tree = renderer.create(<PostPageRecieverHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
