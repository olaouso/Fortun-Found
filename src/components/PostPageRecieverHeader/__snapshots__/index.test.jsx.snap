// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PostPageRecieverHeader component correctly 1`] = `
<div
  className="postRecieveHeader container-md"
>
  <div
    className="row"
  >
    <div
      className="col-md-6"
    >
      <img
        alt="postPageReciever.altRecieverHeader"
        className="recieverHeaderImage"
        src="recieverHeaderPic.jpg"
      />
    </div>
    <div
      className="postHeaderRecieverText col"
    >
      <p>
        postPageReciever.title
      </p>
    </div>
  </div>
</div>
`;
